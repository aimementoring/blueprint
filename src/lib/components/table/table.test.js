import React from 'react';
import renderer from 'react-test-renderer';
import Table from './table.js';

describe('Table', () => {
	it('renders properly', () => {
		const tree = renderer
			.create(
				<Table />,
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
