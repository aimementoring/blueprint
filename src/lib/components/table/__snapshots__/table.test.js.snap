// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table renders properly 1`] = `
<div
  className="theme-portal"
>
  <div
    className=""
  >
    <div
      className="tableContainer"
    >
      <div
        className="rc-table rc-table-scroll-position-left"
        style={Object {}}
      >
        <div
          className="rc-table-content"
        >
          <div
            className="rc-table-body"
            onScroll={[Function]}
            onWheel={[Function]}
            style={Object {}}
          >
            <table
              className=""
              style={Object {}}
            >
              <colgroup />
              <thead
                className="rc-table-thead"
              />
              <tbody
                className="rc-table-tbody"
              />
            </table>
          </div>
          <div
            className="rc-table-placeholder"
          >
            No data available
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
